client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/unified-design-picker.tsx

```diff
@@ -151,6 +151,8 @@ const UnifiedDesignPickerStep: Step = ( { navigation, flow } ) => {
 	const isEnabledStyleSelection =
 		selectedDesign &&
 		selectedDesign.design_type !== 'vertical' &&
+		selectedDesign.style_variations &&
+		selectedDesign.style_variations.length > 0 &&
```

Comment:
Do we need to show the style variations even if the design has only 1 variation?

Reply:
The default style isn't counted as a variation, so when a theme only has the default style, then `style_variations` will be empty (`[]`). When a theme has style variations, `style_variations` returns default style + variation styles. 

So technically, `style_variations` will either be empty, or having two styles or more. But not one.

Reply:
Got it 👍